src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import './Services/Session_id/Session_id'
```

Comment:
CamelCase or SnakeCase?

Reply:
I'll use CamelCase in all cases. (Changed Session_id for sessionId)